### PR TITLE
Add new return value for service queries

### PR DESCRIPTION
**Pull request by NikitaSkrynnik**

### CODE REVIEW

**Review by denis-tingaikin — APPROVED** [review]

**Review by edwarnicke — APPROVED** [review]

**Review by Bolodya1997 — APPROVED** [review]

**Review by glazychev-art — APPROVED** [review]

File: pkg/api/registry/helpers.go [hunk 1]
```diff
@@ -100,3 +104,13 @@ func (x *NetworkService) Clone() *NetworkService {
 func (x *NetworkServiceEndpoint) Clone() *NetworkServiceEndpoint {
 	return proto.Clone(x).(*NetworkServiceEndpoint)
 }
+
+// Clone clones request
```
Comment (denis-tingaikin) [inline pkg/api/registry/helpers.go:108]:
```suggestion
// Clone clones NetworkServiceResponse
```

File: pkg/api/registry/helpers.go [hunk 2]
```diff
@@ -100,3 +104,13 @@ func (x *NetworkService) Clone() *NetworkService {
...
+func (x *NetworkServiceResponse) Clone() *NetworkServiceResponse {
+	return proto.Clone(x).(*NetworkServiceResponse)
+}
+
+// Clone clones request
```
Comment (denis-tingaikin) [inline pkg/api/registry/helpers.go:113]:
```suggestion
// Clone clones NetworkServiceResponse
```

File: pkg/api/registry/registry.pb.go [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
-// 	protoc        v3.14.0
+// 	protoc-gen-go v1.27.1
+// 	protoc        v3.6.1
```
Comment (denis-tingaikin) [inline pkg/api/registry/registry.pb.go:4]: I'm not sure that we should change these versions.

@NikitaSkrynnik Could you re-generate it with 
```
// 	protoc-gen-go v1.26.0
// 	protoc        v3.14.0
```?

**Comment by denis-tingaikin** [comment]
@NikitaSkrynnik Thanks!

**Pull request merged**